emscripten.py

```diff
@@ -1411,7 +1411,7 @@ def asmjs_mangle(name):
         metadata['declares'].append(func_name)
     elif line.startswith('  (func '):
       parts = line.split(' ')
```

Comment:
Did that and changed the nearby branches to do the same, because that seems sane.